src/main/java/javax/jmdns/impl/JmmDNSImpl.java

```diff
@@ -85,11 +87,20 @@
 
     private final AtomicBoolean                                _closed;
 
+    private String                                              _defaultHostName = null;
+
     /**
      *
      */
     public JmmDNSImpl() {
         super();
+
+        try {
+            _defaultHostName = InetAddress.getLocalHost().getHostName();
```

Comment:
Thanks for your response. I will try another way to set the same name for all JmDNS instances of the same computer.